make/go.mk

```diff
@@ -43,6 +47,9 @@ clean-metadata:
 generate-metadata: clean-metadata
 	@echo "generating templates metadata for manifests in $(NSTEMPLATES_BASEDIR)" 
 	@$(foreach tmpl,$(wildcard $(NSTEMPLATES_FILES)),$(call git_commit,$(tmpl),$(NSTEMPLATES_BASEDIR),metadata.yaml))
+	@echo "generating templates metadata for manifests in $(USERTEMPLATES_BASEDIR)" 
+	@rm ./$(USERTEMPLATES_BASEDIR)/metadata.yaml 2>/dev/null || true
+	@$(foreach tmpl,$(wildcard $(USERTEMPLATES_FILES)),$(call git_commit,$(tmpl),$(USERTEMPLATES_BASEDIR),metadata.yaml))
```

Comment:
we don't need to version the UserTiers, right? or do we need that?

Reply:
ah that's right, we don't have any references so it's not necessary... 🤦

Reply:
Removed: https://github.com/codeready-toolchain/host-operator/pull/632/commits/ec8bb521c62159dee5097715788a22cf7ce4282d